index.js

```diff
@@ -10,26 +10,9 @@ function ask(questionText) {
 start();
 
 async function start() {
-
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.");
-  let min = 1;
-  let max = 100;
-  let guess = (Math.floor(Math.random() * (max - min + 1)) + min);
-  
-  let compQuestion = await ask("Is your secret number: " + guess + "?" + "\n(Y)es or (N)o \n"); {
-  
-  console.log(compQuestion);
-    while (compQuestion === "N") {
```

Comment:
You could easily double the number of allowable outputs by checking if `compQuestion.toUpperCase() === "N"` this will allow you to use lowercase letters for your input and it won't break the program.